sunpy/net/vso/attrs.py

```diff
@@ -64,40 +63,6 @@ def collides(self, other):
         return isinstance(other, self.__class__)
 
 
-class Provider(_attr.SimpleAttr):
```

Comment:
You need to add these to the list of removed attrs at the bottom of this file.

Reply:
Done